### ISSUE TITLE
Not displaying red badge for unread messages in Office 365 Outlook Web App

### ISSUE THREAD

**Issue by gitanoqevaporelmundoentero** [meetfranz/franz#1247]
When receiving new unread mails/messages in my configured Office 365 Outlook Web App service, NO red badge icon is displayed for this service. The service properly detects incoming mails and displays corresponding system notification, but NO red badge icon is displayed.

I'm using Franz in my Desktop with following configuration:
 - OS: Ubuntu 18.04
 - Browser: Chrome  72.0.3626.81 (Official Build) (64-bit)

**Comment by mjchamplin** [comment]
+1 to this. When switched to "New Outlook" notifications aren't working and the calendar opens in a new window.

**Comment by tchiotludo** [comment]
I have the same issue. But when I go back to the old ui, It also don't work. 
No more notification for outlook 365

**Comment by dhc02** [comment]
I went back to the "Old Outlook" UI and my Franz notifications came back. (OSX)

**Comment by Nicolas-Medina-Teracode** [comment]
What about the desktop's notifications? I can't manage to see them.

**Comment by danielvarcas** [comment]
I'm not getting the badge or any notifications with the new outlook.

**Comment by jeanfrancisroy** [comment]
There is no such option to switch to the old Outlook, making notifications definitely not working :(

**Comment by jeanfrancisroy** [comment]
I looked at how Franz actually looks for the number of unread messages, and I hacked a way to fix it. I'm no javascript expert so I know that it's not the best (I get the text of a div and split it around), but heh, it works.

First, open `.config/Ferdi/recipes/office365-owa/webview.js`

Then change the `const unreadMail` for the following atrocity:
`    const unreadMail = parseInt(jQuery("div[title*='Inbox']").first().text().split("Inbox")[1].split("unread")[0], 10);
`

**Comment by strigona-worksight** [comment]
Not sure if it's any better, but I took a kick at the can after seeing your comment:
`const unreadMail = parseInt(jQuery("div[title*='Inbox'] > span > span").first().contents().first().text(), 10);`

**Comment by jeanfrancisroy** [comment]
@strigona-worksight it looks better, thanks! I never used jQuery before trying to fix this ;-)

**Comment by strigona-worksight** [comment]
I'm far from a jQuery expert. I tracked down the repo for this recipe and there's an outstanding PR to get this fixed: https://github.com/adambirds/recipe-office365-owa/pull/10

**Comment by ChrisRomp** [comment]
Yeah pretty sure the author isn't updating it since my PR there is almost a year old. Maybe this is due for a takeover.  I see your note on my PR there @strigona-worksight - I'll apply your fix to my PR at least. 🤷‍♂

**Comment by ChrisRomp** [comment]
> Latest commit 254f289 on Oct 23, 2017

Yeah....

**Comment by strigona-worksight** [comment]
Ahh I have spread the conversation all over the place.. sorry. I have submitted an updated fork to Franz: https://github.com/meetfranz/plugins/issues/435

**Comment by lynnedob** [comment]
try this...
https://acompli.helpshift.com/a/outlook-mac/?s=troubleshooting&f=i-am-not-getting-new-email-notifications-after-upgrading-to-macos-catalina

**Comment by MarritS** [comment]
I have the same problem with Windows 10 OS
